teps/0102-https-connection-to-triggers-interceptor.md

```diff
@@ -0,0 +1,85 @@
+---
+title: HTTPS Connection to Triggers ClusterInterceptor
+authors:
+  - "@savitaashture"
+creation-date: 2022-03-21
+last-updated: 2022-03-21
+status: proposed
+---
+
+# TEP-0102: HTTPS Connection to Triggers ClusterInterceptor
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+- [Design Details](#design-details)
+- [Implementation Details](#implementation-details)
+- [A look into the future](#a-look-into-the-future)
+- [References](#references)
+<!-- /toc -->
+
+## Summary
+
+This TEP proposes to run ClusterInterceptor server as HTTPS and to provide a secure connection between 
+EventListener and ClusterInterceptor. 
+
+## Motivation
+
+Running ClusterInterceptor server as HTTPS provide a secure connection to eventlistener
+and also helps triggers users where there is a hard requirement to handle all connections securely.
+
+### Goals
+
+* Running ClusterInterceptor as `HTTPS` and making sure a secure connection between Eventlistener and ClusterInterceptor.
+* No configuration changes asked from user.
+
+### Non-Goals
+
+* Requiring inputs from end user
+
+## Proposal
+
+Triggers now have full support of end to end secure connection by Running ClusterInterceptor as `HTTPS`.
+Triggers uses knative libraries to generate self signed certs in the same way that knative/pkg uses it for admission webhooks.
+
+### User Stories
+
+* ClusterInterceptor calls are done using `HTTP` instead of `HTTPS` which is considered a security problem because
```

Comment:
I'd say this makes more sense in the Motivation section than in the User Stories  section :)